src/llmcompressor/pipelines/sequential/helpers.py

```diff
@@ -85,15 +89,27 @@ def trace_subgraphs(
     :return: a list of Subgraphs in order of execution
     """
     # find modules
-    sequential_targets = match_modules(model, sequential_targets)
-    ignore = match_modules(model, ignore)
+    ignore = ["_update_causal_mask"]
```

Comment:
the `ignore` passed into `trace_subgraphs` is no longer used?

Reply:
I've added `DatasetArguments.tracing_ignore` which is configurable, defaults to `"_update_causal_mask"`